std/haxe/macro/CompilationServer.hx

```diff
@@ -69,6 +69,16 @@ class CompilationServer {
 		});
 	}
 
+	/**
+		Invalidates a module, removing it from the cache.
+
+		If the module has already been loaded in current context, a compiler
+		error will be raised which can be caught using `try ... catch`.
```

Comment:
This should probably mention what to actually catch.

Reply:
I'm not against it, but we also don't mention it on other cases (`Context` API for example).